src/main/java/uk/gov/service/notify/Notification.java

```diff
@@ -21,16 +21,29 @@ public class Notification {
     private String postage;
     private String notificationType;
     private String status;
-    private UUID templateId;
-    private int templateVersion;
-    private String templateUri;
     private String body;
     private String subject;
     private ZonedDateTime createdAt;
     private ZonedDateTime sentAt;
     private ZonedDateTime completedAt;
     private ZonedDateTime estimatedDelivery;
+//     private ZonedDateTime scheduledFor;
+//     private String oneClickUnsubscribe;
     private String createdByName;
+    private boolean isCostDataReady;
+    private double costInPounds;
+
+    // Template fields
+    private UUID templateId;
+    private int templateVersion;
+    private String templateUri;
+
+    // CostDetails fields
+    private Integer billableSmsFragments;
+    private Double internationalRateMultiplier;
+    private Double smsRate;
+    private Integer billableSheetsOfPaper;
+    private String costPostage;
```

Comment:
Is there a reason this is costPostage? I think postageType or postageRate might be better if it is "first class" but if it is for consistency, then that is okay.

Reply:
I didn't know what to call it really.
The field name is `postage` but the Notification class already has a postage field too, and since we use a flat structure, the names would clash. I needed to make a distinction between the postage from Notification and nested postage from CostDetails.

I like postageType so I'll change to it

Reply:
https://github.com/alphagov/notifications-java-client/pull/244/commits/975e56c85caac90d464d2e1a4e136bfce38fbb3c

Reply:
I also updated the documentation: https://github.com/alphagov/notifications-tech-docs/pull/243/commits/578dc7fc940f6713566b1a446b0a610c0600c55c